Add steps to remove invalid or expired invite (#4292)
segmentio/segment-docs: src/segment-app/iam/membership.md

If you are a [workspace member](/docs/segment-app/iam/roles), you might encounter a section of the Segment App that you do not have access to view. If you need expanded permissions, you can request access directly in the Segment App. Once submitted, Access requests are sent to all workspace owners by email.

To review an access request, workspace owners click the link in the access request email to go to their workspace's Access Management Settings. The requestor's access request message appears on the Segment Access Management page, and the workspace owner can adjust the user's permissions. The access request message disappears after the permissions are updated.

## Remove invalid or expired Invite

> When you send an invitation to an incorrect email address or the token included in the email invite link expires, the invite might still show up as "Invite Pending" in Segment App. In these cases, you can revoke the invite to remove it and resend the invite if needed. The invitation will expire within a few days. Therefore, if the user accepts the invite containing an expired link, that user will not be granted any accesses. 

Reviewer: ```suggestion
When you send an invitation to an incorrect email address or the token included in the email invite link expires, the invite might still show up as "Invite Pending" in the Segment App. In these cases, you can revoke the invite to remove it and resend the invite if needed. The invitation will expire within a few days. Therefore, if a user accepts an invite with an expired link, Segment does not grant them access. 
```
